index.html

```diff
@@ -186,7 +186,7 @@ <h2>Glossary</h2>
 
 <p><dfn class="lint-ignore export">Character set</dfn> or <span class="alt_name">repertoire</span>. The set of characters one might use for a particular purpose – be it those required to support Western European languages in computers, or those a Chinese child will learn at school in the third grade (nothing to do with computers).</p>
 
-<p><dfn class="lint-ignore export">Circumgraph</dfn>. A single vowel code point that produces glyphs on more than one side of its consonant base. For example, in the Odia syllable <img src="img/circumgraph.svg" alt="କୋ" style="height:1.2rem;">, /ka/, the character <span class="codepoint" translate="no"><bdi lang="or">&#x0B4B;</bdi> [<span class="uname">U+0B4B ORIYA VOWEL SIGN O</span>]</span> produces separate glyphs on either side of the base consonant.</p>
+<p><dfn class="lint-ignore export">Circumgraph</dfn>. A single vowel code point that produces glyphs on more than one side of its consonant base. For example, in the Odia syllable <img src="img/circumgraph.svg" alt="କୋ" width=33 height=15>, /ka/, the character <span class="codepoint" translate="no"><bdi lang="or">&#x0B4B;</bdi> [<span class="uname">U+0B4B ORIYA VOWEL SIGN O</span>]</span> produces separate glyphs on either side of the base consonant.</p>
```

Comment:
This seems to work. But I still find my experience with using (illegal) `rem` values in the attributes to be weird. Is there a gap in HTML's documentation of "what browsers do"?

Is the source of the problem here the CSS `height` style? Does that activate the autosizing in the browsers maybe?

Reply:
I suppose we could have two solution here:

- (as suggested) removing style and adding `width=` and `height=`, to prevent automatic addition of width/height by ReSpec
- changing style with `height: 1.2rem; width: auto;`, and let ReSpec do anything (adding width/height for Chrome, nothing for Firefox)

I am quite not sure whether Chrome's behavior of just overriding height by CSS with *keeping aspect ratio* specified by height and width HTML attribute, but having `width: auto` will make `img` to be scaled matching with the original aspect ratio.

Reply:
A third option is to stop Respec adding these attributes in the first place, because they set fixed pixel sizes that we'd have to change manually when we want to change the styling for the glossary (eg. larger font size or line height) in future.  I added a comment to the respec issue that @xfq found.

Reply:
FWIW, in clreq, I added explicit width and height to all images to reduce layout shifts according to [this article](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/).